Assets/HoloToolkit/Utilities/Scripts/CameraCache.cs

```diff
@@ -20,6 +20,7 @@ public static Camera Main
         {
             get
             {
+                // ReSharper disable once ConvertIfStatementToReturnStatement
```

Comment:
I'm unsure if we should include ReSharper stuff in the project.

Reply:
Ok